src/ticks.cpp

```diff
@@ -28,10 +28,9 @@
 
 #define __STDC_LIMIT_MACROS // for INT64_MAX in __cplusplus
 
+#include <stdio.h>
```

Comment:
any benefit from this change?

Reply:
When I compiled on windows there was a problem with order of including library. 
Changing the order solved this..